HACKING.md

```diff
@@ -33,6 +35,37 @@ Follow the instructions to install the operator, e.g.:
 $ oc apply -f /tmp/manifests/path
 ```
 
+#### Building Remotely on the Cluster
+
+To build the operator on the remote cluster, first create a buildconfig.  Use the following command, with the appropriate strings substituted for `<username>` and `<branch>`:
+
+```
+$ oc process -f hack/buildconfig.yaml \
+     -p 'GIT_URL=https://github.com/<username>/cluster-ingress-operator.git' \
+     -p 'GIT_BRANCH=<branch>' \
+      | oc -n openshift-ingress-operator create -f -
+```
+
+Next, start a build from this buildconfig:
+
+```
+$ oc -n openshift-ingress-operator start-build cluster-ingress-operator --follow
+```
+
+The build will build a new image and push it to the cluster's internal registry. Once the build is done, patch the operator to use the newly built image:
+
+```
+oc -n openshift-ingress-operator patch deploy/ingress-operator --type=json \
+   --patch='
+- op: replace
+  path: /spec/template/spec/containers/0/image
+  value: image-registry.openshift-image-registry.svc:5000/openshift-ingress-operator/cluster-ingress-operator:latest
+- op: replace
+  path: /spec/template/spec/containers/0/imagePullPolicy
+  value: Always
+'
+```
```

Comment:
The theoretical `make build-cluster` could handle this, too (if, say, `DEPLOY=true` or something).

Reply:
Done (`make cluster-build DEPLOY=1`).